workspace/libs/shared/ui/src/lib/shared-ui.module.ts

```diff
@@ -80,6 +81,7 @@ import { SafePipe } from './pipes/safe.pipe';
     PostalPipe,
     ReplacePipe,
     SafePipe,
+    HtmlComponent,
```

Comment:
Lets try and keep the components, pipes, etc; that are declared grouped in the imports and exports